src/store/configureStore.js

```diff
@@ -50,7 +51,11 @@ function configureStore(initialState: $Shape<AppState> = {}, history: Object) {
   // ======================================================
   // Run sagas
   // ======================================================
-  Object.keys(sagas).forEach(sagaName => sagaMiddleware.run(sagas[sagaName]))
+  forOwn(sagas, saga =>
+    isObject(saga) ?
+      forOwn(saga, sagaMiddleware.run)
+      : sagaMiddleware.run(saga)
+  )
```

Comment:
better to avoid unrelated changes

Reply:
It's not unrelated changes, because sagas are exporting other way now